routes/conversation.js

```diff
@@ -54,11 +54,9 @@ router.put("/:conversationId", async function (req, res) {
         });
     }
 
-    conversation = Conversation.findById(req.params.conversationId);
-    conversation.messages.add(message);
-    await conversation.save();
+    await Conversation.updateOne({_id: req.params.conversationId}, {$push: {messages: req.body.message}});
```

Comment:
Push the user ID alongside the message so we know who sent what message

Reply:
The message object itself contains userId, message (text) and userName as per schema, open to being more specific on the validation of empty fields if needed.